docs/RHEL Qcow2 to OVA.md

```diff
@@ -0,0 +1,26 @@
+# Overview
+This guide talks about how to convert the CentOS qcow2 to ova format
```

Comment:
The example shows RHEL qcow2 to ova, so please change the overview statement.

Reply:
will fix this